goss.yaml

```diff
@@ -0,0 +1,16 @@
+command:
```

Comment:
Can we use cli args here somehow, specifically model name, to pass into `juju -m {model_name}`?